starlite/logging/__init__.py

```diff
@@ -49,5 +54,24 @@ class LoggingConfig(BaseModel):
     except that the propagate setting will not be applicable."""
 
     def configure(self) -> None:
-        """Configured logger with the given configuration."""
+        """Configured logger with the given configuration.
+
+        If the logger class contains the word `picologging`, we try to
+        import and set the dictConfig
+        """
+        for logging_class in _find_keys(self.handlers, "class"):
+            if "picologging" in logging_class and picologging_config:
+                picologging_config.dictConfig(self.dict(exclude_none=True))
+                break
         config.dictConfig(self.dict(exclude_none=True))
+
+
+def _find_keys(node: Union[List, Dict], key: str) -> Any:
```

Comment:
Missing docstring. Also typing should be Generator[Iterable, None, None] if I understand the code correctly